packages/cmr-client/deleteConcept.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const Logger = require('@cumulus/logger');
+const got = require('got');
+const { parseString } = require('xml2js');
+const { promisify } = require('util');
+
+const getUrl = require('./getUrl');
+
+const log = new Logger({ sender: 'cmr-client' });
+// TODO copied in ingestConcept
+const xmlParseOptions = {
```

Comment:
Since these options are only ever used in `xml2js.parseString()`, you could DRY this up by creating a `parseString()` function of your own that the other files in `cmr-client` just import.  That would be a good place to handle promisifying `parseString()` as well.